trackmania2020/egg-trackmania2020.json

```diff
@@ -0,0 +1,150 @@
+{
+    "_comment": "DO NOT EDIT: FILE GENERATED AUTOMATICALLY BY PANEL",
+    "meta": {
+        "version": "PLCN_v2",
+        "update_url": null
+    },
+    "exported_at": "2025-09-27T16:42:32+00:00",
+    "name": "Trackmania (2020)",
+    "author": "pelican@stather.me",
+    "uuid": "a8135398-e0cb-44fd-93fc-e84564febdb1",
+    "description": "Trackmania is a free-to-play game that combines racing and creation. Compete on over 1000 community-made tracks, create your own with the editor, and join clubs and events online.",
+    "tags": [],
+    "features": [],
+    "docker_images": {
+        "ghcr.io/parkervcp/yolks:debian": "ghcr.io/parkervcp/yolks:debian"
+    },
+    "file_denylist": [],
+    "startup": "./TrackmaniaServer /nodaemon /dedicated_cfg=dedicated_cfg.xml /game_settings={{MATCH_SETTINGS}} /servername=\"{{SERVER_NAME}}\"",
+    "config": {
+        "files": "{\n    \"UserData/Config/dedicated_cfg.xml\": {\n        \"parser\": \"xml\",\n        \"find\": {\n            \"dedicated.server_options.name\": \"{{server.environment.SERVER_NAME}}\",\n            \"dedicated.server_options.password\": \"{{server.environment.SERVER_PASSWORD}}\",\n            \"dedicated.server_options.password_spectator\": \"{{server.environment.SPECTATOR_PASSWORD}}\",\n            \"dedicated.server_options.max_players\": \"{{server.environment.MAX_PLAYERS}}\",\n            \"dedicated.server_options.max_spectators\": \"{{server.environment.MAX_SPECTATORS}}\",\n            \"dedicated.system_config.server_port\": \"{{server.allocations.default.port}}\",\n            \"dedicated.system_config.xmlrpc_port\": \"{{server.environment.XMLRPC_PORT}}\",\n            \"dedicated.masterserver_account.login\": \"{{server.environment.MASTER_SERVER_LOGIN}}\",\n            \"dedicated.masterserver_account.password\": \"{{server.environment.MASTER_SERVER_PASSWORD}}\"\n        }\n    }\n}",
+        "startup": "{\"done\": \"...Load succeeds\"}",
+        "logs": "{}",
+        "stop": "^C"
+    },
+    "scripts": {
+        "installation": {
+            "script": "#!/bin/bash\r\n# Server files: /mnt/server\r\nmkdir -p /mnt/server\r\ncd /mnt/server\r\n\r\nwget http://files.v04.maniaplanet.com/server/TrackmaniaServer_Latest.zip\r\nunzip -o TrackmaniaServer_Latest.zip\r\nrm TrackmaniaServer_Latest.zip\r\nchmod 755 TrackmaniaServer\r\n\r\ncp -n UserData/Config/dedicated_cfg.default.txt UserData/Config/dedicated_cfg.xml\r\ncp -n UserData/Maps/MatchSettings/example.txt  UserData/Maps/matchsettings.xml",
```

Comment:
add and echo print at the end that it is done installing

Reply:
Added an echo at the end